web/src/components/EventListItem.tsx

```diff
@@ -28,10 +28,10 @@ const getEventPlaceholder = (path: string): string => {
 const EventListItem = ({ event, formatter }: PropsType): ReactElement => (
   <ListItem thumbnail={event.thumbnail || getEventPlaceholder(event.path)} title={event.title} path={event.path}>
     <div>
-      <div>{event.date.toFormattedString(formatter)}</div>
-      {event.location && <div>{event.location.fullAddress}</div>}
+      <div dir='auto'>{event.date.toFormattedString(formatter)}</div>
```

Comment:
Hmh, would it make sense to create a component for this, which we reuse?

Reply:
Thought about this as well but there are quite a few different html elements we set this property on so, so separate reusable components or casts would be necessary so I decided I'll just keep it this way. Let me know if you'd prefer reusable component(s).

Reply:
Fine for me. We can change it in the future with same effort anyways, if we change our opinion.